src/audio/buffers/audio_buffer.c

```diff
@@ -95,3 +96,128 @@ void audio_buffer_free(struct sof_audio_buffer *buffer)
 		buffer->ops->free(buffer);
 	rfree(buffer);
 }
+
+static
```

Comment:
This "static on its own line" is not matching existing style we've used in SOF.

Reply:
it's a helper for too long lines...